docs/Card/README.md

```diff
@@ -144,9 +144,15 @@ Optionally, a merchant app can request Strong Consumer Authentication (SCA) for
 To request SCA, add the following to `CardRequest`:
 
 ```swift
-cardRequest.threeDSecureRequest = ThreeDSecureRequest(sca: .always)
+cardRequest.threeDSecureRequest = ThreeDSecureRequest(
+    sca: .always,
+    returnUrl: "\(Bundle.main.bundleIdentifier)://return_url",
+    cancelUrl: "\(Bundle.main.bundleIdentifier)://cancel_url"
+)
 ```
 
+Where `return_url` and `cancel_url` are the urls configured in your [paypal developer dashboard](https://developer.paypal.com/)
```

Comment:
I remember Jon mentioned that this was no longer a requirement, for merchants to need to upload their URLS to the dev dashboard. Do we know which version this requirement was dropped in?

Reply:
@scannillo i think Jon meant for that for NXO. If i dont put it here, i get an error on the browser.